traffic_portal/package.json

```diff
@@ -33,5 +33,10 @@
   },
   "engines": {
     "node": "0.11.14"
+  },
+  "dependencies": {
```

Comment:
is this really needed? not sure why it would be when i saw that you included the js files directly in the codebase